explorer/client/src/utils/internetapi/TransactionTypes.ts

```diff
@@ -0,0 +1,78 @@
+
+import { AddressBytes, AuthoritySignInfo, GasPayment, Signature, TransactionKind } from "./SuiRpcClient"
+
+type SuiBytes = number[] | string;
+
+type TransactionStatus = string;
+type TransactionDigest = SuiBytes;
+type ObjectRef = SuiBytes;
+type Owner = SuiBytes;
+type ObjectRefOwnerPair = [ObjectRef, Owner]
+
+// many types in this file map directly to Rust types of the same name
+type TransactionEffects = {
+    status: TransactionStatus,
+    digest: TransactionDigest,
+    created?: ObjectRefOwnerPair[],
+    mutated?: ObjectRefOwnerPair[],
+    unwrapped?: ObjectRefOwnerPair[],
+    deleted?: ObjectRef[],
+    wrapped?: ObjectRef[],
+    gas?: ObjectRefOwnerPair,
+    events?: any[],                     // TODO - better type
+    dependencies?: TransactionDigest[]
+}
+
+interface TransactionEffectsEnvelope<S> {
+    effects: TransactionEffects,
+    auth_signature?: S
+}
+
+type AuthorityName = string;
+type AuthorityNameSigPair = [AuthorityName, SuiBytes];
+
+type TransactionEnvelope<S> = {
+    is_checked: boolean,
```

Comment:
`is_checked` is marked as `skip` [here](https://github.com/MystenLabs/sui/blob/f9ef1783083bc5011951ae3a779bb70d27cb8335/sui_types/src/messages.rs#L333), so it seems that this field won't be sent to the client side?